### PR TITLE
storage: add stat for number of Raft commands applied

### PR DESCRIPTION
**Pull request by spencerkimball**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/12488)

### CODE REVIEW

**Review by a-robinson — APPROVED** [review]
LGTM

File: pkg/storage/metrics.go [hunk 1]
```diff
@@ -389,6 +395,7 @@ type StoreMetrics struct {
 	RaftTicks                *metric.Counter
 	RaftWorkingDurationNanos *metric.Counter
 	RaftTickingDurationNanos *metric.Counter
+	RaftCommandsApplied      *metric.counter
```
Comment (a-robinson) [inline pkg/storage/metrics.go]: s/counter/Counter

**Comment by bdarnell** [comment]
:lgtm:

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/12488#-:-KZQ4vGqM0kM8h0lRCvC:bnfp4nl)*

**Review by petermattis — APPROVED** [review]
LGTM (once it compiles)

**Pull request merged**